src/jobs/pack.yml

```diff
@@ -21,10 +26,24 @@ steps:
         ORB_PARAM_SOURCE_DIR: << parameters.source-dir >>
         ORB_PARAM_OUTPUT_DIR: << parameters.output-dir >>
       command: <<include(scripts/pack.sh)>>
+  - when:
```

Comment:
Love this, smart use of YAML conditionals.

unfortunately the in-line bash on line 37 will fail the RC check which requires that code be stored in a shell file.

My recommendation. Remove the YAML conditional and this extra step, and put all of the logic directly in the shell `validate` shell script. Bash `if` statements can be used rather than YAML conditioning. This will also result in less steps in the UI.

Reply:
thanks for the heads-up on the RC check.
I have moved the conditional checking all into the validate.sh script in https://github.com/CircleCI-Public/orb-tools-orb/pull/161/commits/0e7b77bfb950dd88320474a446a677470abb040c, as suggested then 👍